prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
+    </script>
```

Comment:
this code, minus the script tag, should be in the main.js file

Reply:
<script src="main.js" console.log('I have added a script file via the src attribute!');></script>
 
Is this correct? I'm not sure how to place the console.log code "in" the main.js file...